src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+
+  checHealth() {
```

Comment:
There is a typo in the method name `checHealth`. It should be `checkHealth` to accurately describe its functionality and to avoid confusion.